edit-post/components/sidebar/featured-image/index.js

```diff
@@ -21,17 +24,21 @@ import { toggleSidebarPanel } from '../../../store/actions';
  */
 const PANEL_NAME = 'featured-image';
 
-function FeaturedImage( { isOpened, onTogglePanel } ) {
+function FeaturedImage( { isOpened, postType, onTogglePanel } ) {
 	return (
 		<PostFeaturedImageCheck>
-			<PanelBody title={ __( 'Featured Image' ) } opened={ isOpened } onToggle={ onTogglePanel }>
+			<PanelBody title={ get( postType, 'data.labels.featured_image', __( 'Featured Image' ) ) } opened={ isOpened } onToggle={ onTogglePanel }>
```

Comment:
This line looks too long, can we do 1 prop per line?

Reply:
Updated 👍